### PR TITLE
hotfix: update knex dependency to resolve security warning

### PR DESCRIPTION
**Pull request by sitozzz**

### CODE REVIEW

**Review by toplenboren — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -90,7 +90,7 @@
     "es5-ext": "0.10.53",
     "debuglog": "1.0.1",
     "debug": "4.3.4",
-    "knex": "0.95.15",
+    "knex": "2.4.0",
```
Comment (toplenboren) [inline package.json:93]: We also need to double check `adapterCache` with this change

**Review by toplenboren — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sitozzz) [reply package.json:93]: Yup, that's a great suggest. But do we have tests of the caching mechanism?

**Review by SavelevMatthew — APPROVED** [review]
🤞

File: package.json (hunk 1 above)
Comment (toplenboren) [reply package.json:93]: yea

**Pull request merged**